packages/graphql-api/src/utils/authenticated-resolver.ts

```diff
@@ -1,4 +1,8 @@
 export const authenticated = (Accounts, func) => (async (root, args, context, info) => {
+  if (context && context.skipJSAccountsVerification === true) {
```

Comment:
Why is this good? I would expect users that does not want to use js-accounts verification to just not use the authenticated wrapper no?

Reply:
Because in some cases you want to skip the validation. 
For example: I am using the GraphQL execute as standalone in the server side, without the client executing anything, and I would like to execute it with a specific user.
Without the `skipJSAccountsVerification`, the JSAccounts will override it.
I want to be able to execute the specific schema with and without a user, but still to use `authenticated`